src/app/contexts/ToggleContext/reducer.jsx

```diff
@@ -17,7 +16,7 @@ export const toggleReducer = (toggleState, action) => {
         ...toggleState,
         ...data.toggles,
         ads: {
-          enabled: adsEnabled(data),
+          enabled: adsEnabled({ data }),
```

Comment:
Original version: `return { ...toggleState, ...data.toggles };`

https://github.com/bbc/simorgh/pull/6287/files#diff-62c1f6347f51f748dcf67aaaf6e68e33

Reply:
@mmcculloch Are you suggesting we remove the check for toggles.ads.enabled in this PR?